docs/cookbook/admin_panel/menu.md

```diff
@@ -88,3 +88,18 @@ final readonly class MenuBuilder implements MenuBuilderInterface
     }
 }
 ```
+
+{% hint style="success" %}
+**🧠 Collapse your custom menu by default**
+
+It's possible to expand your parent menu category on page load by default. For that, you have to set the `setExtra` attribute like this:
+
+```php
+$newSubmenu = $menu
+    ->addChild('new')
+    ->setLabel('Custom Admin Menu')
+    ->setExtra('always_open', true);
```

Comment:
Since the menu generation example on the Sylius stack docs is different from the one on [docs.sylius.com](https://docs.sylius.com/the-customization-guide/customizing-menus), I would suggest adjusting the example for the Sylius stack docs according to the menu generation example there:

```suggestion
$library = $menu
    ->addChild('library')
    ->setLabel('app.ui.library')
    ->setLabelAttribute('icon', 'tabler:books')
    ->setExtra('always_open', true);
```

Just to avoid possible confusion 😄